specification/translation/Azure.AI.DocumentTranslation/tspconfig.yaml

```diff
@@ -16,19 +16,19 @@ options:
     examples-directory: "{project-root}/examples"
     output-file: "{azure-resource-provider-folder}/Azure.AI.DocumentTranslation/{version-status}/{version}/openapi.json"
   "@azure-tools/typespec-csharp":
-    package-dir: "Azure.AI.Translator.Document"
-    namespace: Azure.AI.Translator.Document
+    package-dir: "Azure.AI.Translation.Document"
+    namespace: Azure.AI.Translation.Document
     clear-output-folder: true
-    model-namespace: false
+    model-namespace: true
   "@azure-tools/typespec-python":    
-    package-dir: "azure-ai-translator-document"
-    package-name: "azure-ai-translator-document"
+    package-dir: "azure-ai-translation-document"
+    package-name: "azure-ai-translation-document"
     package-mode: dataplane
   "@azure-tools/typespec-java":
-    package-dir: "azure-ai-translator-document"
-    namespace: com.azure.ai.translator.document
+    package-dir: "azure-ai-translation-document"
+    namespace: azure-ai-translation-document
   "@azure-tools/typespec-ts":
-    package-dir: "azure-ai-translator-document"
+    package-dir: "azure-ai-translation-document"
```

Comment:
Is the same package with `@azure-rest/ai-document-translator`?

https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/documenttranslator/ai-document-translator-rest